### ISSUE TITLE
Website doesn't work 

### ISSUE THREAD

**Issue by dhalucario** [jamesfmackenzie/chocolatekeen#1]
When I visit the website http://www.jamesfmackenzie.com/chocolatekeen/
the game does not seem to run on my Firefox version.

OS 		Fedora
Name 	Firefox
Version 	106.0.1

All I see is this image:
![image](https://user-images.githubusercontent.com/10533763/201786744-f83a9527-9330-4da0-9818-abaaa7df3a91.png)

**Comment by r0d0dendr0n** [comment]
Yup, the console prints out: `Error: Couldn't initialize SDL: Gamepads not supported`.
